datastore-v1-proto-client/src/test/resources/META-INF/native-image/com/google/datastore/v1/client/reflect-config.json

```diff
@@ -0,0 +1,83 @@
+[
```

Comment:
Minor nit: The json files tend to follow [this naming convention](https://www.graalvm.org/22.0/reference-manual/native-image/BuildConfiguration/#embedding-a-configuration-file): `META-INF/native-image/bar_groupID/bar_artifactID`. 

How about we rename this path to be `.../META-INF/native-image/com.google.cloud.datastore/datastore-v1-proto-client/reflect-config.json`?